jetty-server/src/test/scala/org/http4s/jetty/server/JettyServerSuite.scala

```diff
@@ -80,12 +80,12 @@ class JettyServerSuite extends Http4sSuite {
     }
 
   jettyServer.test("ChannelOptions should route requests on the service executor") { server =>
-    get(server, "/thread/routing").map(_.startsWith("http4s-suite-")).assert
+    get(server, "/thread/routing").map(_.startsWith("io-compute-")).assert
```

Comment:
Changed in https://github.com/typelevel/munit-cats-effect/pull/124.

Reply:
Uh-oh.  I bet my merge from 0.22 into 0.23 has died on this.

Reply:
It did 😛